test/corpus/casting.txt

```diff
@@ -0,0 +1,46 @@
+==================
+Standard cast
+==================
+
+SELECT CAST(1 AS TEXT);
+
+---
+
+(program
+ (statement
+  (select
+    (keyword_select)
+    (select_expression
+      (cast (identifier) (literal) (keyword_as) (keyword_text))))))
+
+==================
+Aliased
+==================
+
+SELECT CAST(1 AS TEXT) AS fieldname;
```

Comment:
Not a blocker for this PR but I do think we'll need a node to enclose an aliased field like this eventually. We can wait for a use case though.